geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -63,14 +66,14 @@
                     {# dropdown is disabled by default; enable if javascript is active #}
                     <details class="itt-select" aria-expanded="false" data-transcription-target="dropdownDetails" data-relation="transcription" data-count="{{ document.digital_editions.count }}" disabled="true">
                         <summary data-action="keydown->transcription#shiftTabCloseDropdown">
-                            <span data-transcription-target="transcriptionShortLabel">Edition: {{ document.digital_editions.0.source.all_authors }}</span>
+                            <span data-transcription-target="transcriptionShortLabel">{% if document.digital_editions.0 %}Edition: {{ document.digital_editions.0.source.all_authors|default:document.digital_editions.0.source }}{% endif %}</span>
```

Comment:
That's right. I didn't think it was worth it since this is the only place where we need this fallback behavior (everywhere else we use the citation), but I can definitely implement if you feel strongly it should be a method.